### PR TITLE
Change to src repo structure

### PR DESCRIPTION
**Pull request by zariiii9003**
With this change, the tests will run on the packaged and installed code. Look at the paths in the coverage summary:

image

It looks like this on the master branch:
image

This should avoid bugs like in #584

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist = py
-isolated_build = true
```
Comment (andlaus) [inline tox.ini:3]: what prevents using isolated builds? (what does the `isolated_build` parameter do in the first place?)

**Review by andlaus — COMMENTED** [review]
in principle, this is okay IMO. That said, be aware that I'd like to significantly flatten the source tree soonish, so it might be worthwhile to do such changes in one go. (maybe not...)

File: tox.ini (hunk 1 above)
Comment (zariiii9003) [reply tox.ini:3]: isolated_build was removed in tox4, the package is always built in a separate venv

**Comment by zariiii9003** [comment]
> in principle, this is okay IMO. That said, be aware that I'd like to significantly flatten the source tree soonish, so it might be worthwhile to do such changes in one go. (maybe not...)

I'd prefer not to change too much in a single PR.

**Comment by andlaus** [comment]
> I'd prefer not to change too much in a single PR.

alright. let's do it piecemeal then...

**Comment by andlaus** [comment]
@zariiii9003: anything which needs to be changed here or can it be merged? (I would like to not keep this open for too long because this has quite a bit of headache potential.)

**Comment by zariiii9003** [comment]
> @zariiii9003: anything which needs to be changed here or can it be merged? (I would like to not keep this open for too long because this has quite a bit of headache potential.)

Done :slightly_smiling_face:

**Pull request merged**